examples/basic-converter/styles.css

```
@@ -1 +1,100 @@
+}
+
+h1 {
+  margin-bottom: 2rem;
+  color: #1a202c;
+  font-size: 2.2rem;
+  width: 100%;
+  max-width: 450px;
+  text-align: left;
+}
+
+pelela {
```

Stylelint (17.9.0)

[error] Unknown type selector "pelela" (selector-type-no-unknown)

(selector-type-no-unknown)